frameworks/PHP/reactphp/server.php

```diff
@@ -1,20 +1,28 @@
 <?php
 
+use React\EventLoop\Loop;
+use React\Http\HttpServer;
+use React\Socket\SocketServer;
+
 require __DIR__ . '/vendor/autoload.php';
 require_once __DIR__.'/app.php';
 
-init();
+$server = new HttpServer(requestHandler());
+$socket = new SocketServer('0.0.0.0:8080');
+$server->listen($socket);
 
-$loop = React\EventLoop\Loop::get();
+echo "React Server running at http://0.0.0.0:8080\n";
+echo "EventLoop: ", Loop::get()::class, "\n";
 
-$server = new React\Http\Server($loop, function (Psr\Http\Message\ServerRequestInterface $request) {
-    return router($request);
-});
+$interrupt = static function () use ($server, $socket, &$interrupt): void {
+    echo 'Interrupting server', PHP_EOL;
 
-$socket = new React\Socket\Server('0.0.0.0:8080', $loop);
-$server->listen($socket);
+    $socket->close();
+    $server->removeAllListeners();
```

Comment:
No need to remove all the listeners, that will be done the moment the socket is closed: 
```suggestion
```